p/themes/base-theme/template.css

```diff
@@ -569,6 +569,12 @@ a.btn {
 	color: #000;
 	text-decoration: none;
 }
+.flux .item.author {
+	color: #555;
+	font-size: 11px;
```

Comment:
It's generally preferable to give a value in `rem` so that everything scales along if you change the top level font size. In a standard configuration that's 11/16 = let's say `.7rem`.

```suggestion
	font-size: .7rem;
```